src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -32,8 +32,9 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
   })
 
   // If there is an error in the old resourceCache, we want to override it
```

Comment:
No, it's still valid. If there's an error, then state.resourceCache.status will be 'error' and hence, we will override it with the new resourceCache without adding anything (because there's nothing to add sth to)